components/dash-core-components/src/components/Input.react.js

```diff
@@ -90,25 +96,58 @@ export default class Input extends PureComponent {
             this.setState({value: this.props.value});
         }
     }
+    formatCurrency(value) {
+        if (isNaN(value) || value === '') {
+            return '';
+        }
+        const {currencySymbol} = this.props;
+
+        return `${currencySymbol}${Number(value).toLocaleString()}`;
```

Comment:
are there cases where the currency symbol comes _after_ the number? Or is that just odd English typography like `15¢` that isn't used for international currencies?